app/src/ui/branches/branches-container.tsx

```diff
@@ -174,42 +173,37 @@ export class BranchesContainer extends React.Component<
 
   private renderPullRequests() {
     const pullRequests = this.props.pullRequests
-    if (pullRequests.length) {
+    const repo = this.props.repository
```

Comment:
:lipstick: move these closer to the component, so it's clear where they're required